fairer-iOS/fairer-iOS/Screens/OnboardingName/OnboardingNameViewController.swift

```diff
@@ -0,0 +1,163 @@
+//
+//  OnboardingNameViewController.swift
+//  fairer-iOS
+//
+//  Created by 김유나 on 2022/09/18.
+//
+
+import UIKit
+
+import SnapKit
+
+final class OnboardingNameViewController: BaseViewController {
+    
+    private let nameMaxLength = 5
+    
+    // MARK: - property
+    
+    private let nameLabel: UILabel = {
+        let label = UILabel()
+        label.text = TextLiteral.onboardingNameViewControllerNameLabel
+        label.textColor = .gray800
+        label.font = .h2
+        return label
+    }()
+    private let nameTextField: UITextField = {
+        let textField = UITextField()
+        let attributes = [
+            NSAttributedString.Key.font: UIFont.h3,
+            NSAttributedString.Key.foregroundColor: UIColor.gray400
+        ]
+        textField.backgroundColor = .normal0
+        textField.font = .h3
+        textField.attributedPlaceholder = NSAttributedString(string: TextLiteral.onboardingNameViewControllerTextFieldPlaceholder, attributes: attributes)
+        textField.layer.cornerRadius = 8
+        textField.autocorrectionType = .no
+        textField.autocapitalizationType = .none
+        textField.leftView = UIView(frame: CGRect(x: 0, y: 0, width: 24, height: 58))
+        textField.leftViewMode = .always
+        textField.setClearButton()
+        return textField
+    }()
+    private let nameDoneButton: MainButton = {
+        let button = MainButton()
+        button.title = TextLiteral.onboardingNameViewControllerDoneButtonText
+        button.isDisabled = true
+        button.addTarget(self, action: #selector(didTapDoneButton), for: .touchUpInside)
+        return button
+    }()
+    private let disableLabel: UILabel = {
+        let label = UILabel()
+        label.text = TextLiteral.onboardingNameViewControllerDisableLabel
+        label.textColor = .negative20
+        label.font = .body2
+        label.layer.opacity = 0
+        return label
+    }()
+    
+    // MARK: - life cycle
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupDelegation()
+        setupNotificationCenter()
+    }
+    
+    override func configUI() {
+        super.configUI()
+    }
+    
+    override func render() {
+        view.addSubview(nameLabel)
+        nameLabel.snp.makeConstraints {
+            $0.leading.equalTo(24)
+            $0.top.equalToSuperview().offset(111)
+        }
+        
+        view.addSubview(nameTextField)
+        nameTextField.snp.makeConstraints {
+            $0.leading.equalTo(24)
+            $0.trailing.equalTo(-24)
```

Comment:
p2;

```suggestion
            $0.leading.trailing. equalToSuperview().inset(24)
``` 
요렇게 쓸 수 있습니다 ^______^

Reply:
오호라 inset으로 하면 leading 때문에 안 밀리는 군요..! 새로 배워 갑니당..

Reply:
[7923939](https://github.com/fairer-iOS/fairer-iOS/pull/15/commits/79239398bf59d8df61c6b592d82fedb78082b477)
수정해서 반영했습니다!